### ISSUE TITLE
Lock down workflow permissions for integration-tests.yml

### ISSUE THREAD

**Issue by dylan-smith** [github/gh-gei#490]
Set the workflow permissions to the minimum required in integration-tests.yml

**Comment by timrogers** [comment]
@dylan-smith Did this get done in the end? 👼

**Comment by dylan-smith** [comment]
I don't think so. We have 3 Actions workflows (CI.yml, integration-tests.yml, and publish-test-results.yml). We are currently only explicitly setting permissions in publish-test-results.yml
